config/routes.rb

```diff
@@ -236,6 +236,7 @@
       put :modules
```

Comment:
@NobodysNightmare @ulferts for consistency's sake, it would probably make sense to use PATCH rather than PUT for these controller actions too. However, since specs are passing for these routes/actions, I'd be loather to make these changes in this PR.